orchestration-alerting/slackNotifications.js

```diff
@@ -0,0 +1,113 @@
+const { getParameter } = require("./aws");
+
+const formatMessage = (snsMessage, colorCode, snsMessageFooter) => {
+    var description = snsMessage.AlarmDescription.split("ACCOUNT:");
+    var account = snsMessage.AWSAccountId;
+    if (description.length > 1) {
+        account = description[1];
+    }
+    if (JSON.stringify(snsMessage).includes("ElastiCache")) {
+        return {
+            attachments: [
+                {
+                    fallback:
+                        Object.keys(snsMessage)[0] +
+                        "for cluster: " +
+                        Object.values(snsMessage)[0],
+                    color: "#ff9966",
+                    title: Object.values(snsMessage)[0] + "-notification",
+                    text:
+                        Object.keys(snsMessage)[0] +
+                        " for cluster: " +
+                        Object.values(snsMessage)[0],
+                    fields: [
+                        {
+                            title: "Status",
+                            value: "INFO",
+                            short: false,
+                        },
+                        {
+                            title: "Account",
+                            value: account,
+                            short: false,
+                        },
+                    ],
+                    footer: snsMessageFooter,
+                },
+            ],
+        };
+    } else {
+        return {
+            attachments: [
+                {
+                    fallback: description[0],
+                    color: colorCode,
+                    title: snsMessage.AlarmName,
+                    text: description[0],
+                    fields: [
+                        {
+                            title: "Status",
+                            value: snsMessage.NewStateValue,
+                            short: false,
+                        },
+                        {
+                            title: "Account",
+                            value: account,
+                            short: false,
+                        },
+                    ],
+                    footer: snsMessageFooter,
+                },
+            ],
+        };
+    }
+};
+
+const buildMessageRequest = (snsMessage, colorCode, snsMessageFooter) => {
+    const body = formatMessage(snsMessage, colorCode, snsMessageFooter);
+    if (process.env.DEPLOY_ENVIRONMENT === "integration") {
```

Comment:
Probably want to add staging here for testing?

Reply:
Ah yes 🤦‍♀️

Reply:
Have added to the """testing""" condition below (as integration will stay on)